python-sdk/tests/extractors/test_extractor.py

```diff
@@ -99,6 +120,53 @@ def test_python_sdk_load_file_extract_on_complete():
     assert task_meta.run_facets == {}
 
 
+@pytest.mark.integration
+def test_python_sdk_export_file_extract_on_complete():
+    """
+    Tests that  the custom PythonSDKExtractor is able to process the
+    operator's metadata that needs to be extracted as per OpenLineage
+    for ExportFileOperator.
+    """
+    LoadFileOperator(
+        task_id="load_file",
+        input_file=File(
+            path="https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb_v2.csv"
+        ),
+        output_table=Table(conn_id="sqlite_conn", name="test_extractor"),
+    ).execute({})
+
+    task_id = "export_file"
+    export_file_operator = ExportFileOperator(
+        task_id=task_id,
+        input_data=Table(conn_id="sqlite_conn", name="test_extractor"),
+        output_file=File(
+            path="gs://astro-sdk/workspace/sample_pattern.csv",
```

Comment:
Can you change this file name please as this file is being used by other tests in the CI as well?

Reply:
yes

Reply:
changed to gs://astro-sdk/workspace/openlineage_export_file.csv